tests/k6/tests/enduser/dialogSystemLabelLog.js

```
@@ -0,0 +1,60 @@
+        let response = putEU('dialogs/' + dialogId + '/systemlabels', body);
+        expectStatusFor(response).to.equal(204);
+        response = getEU('dialogs/' + dialogId + '/labellog');
+        expectStatusFor(response).to.equal(200);
+        expect(response, 'response').to.have.validJsonBody();
+        console.log(response.json());
+        expect(response.json(), 'response body').to.have.lengthOf(3);
+    })
+
+    describe('Dialog update set system label to default', () => {
+        dialog.progress = "60";
+        let response = putSO('dialogs/' + dialogId, dialog);
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)